Tests/SonarQube.MSBuild.Tasks.IntegrationTests/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -226,40 +235,6 @@ public void Roslyn_Settings_ErrorLogAlreadySet()
         }
 
         [TestMethod]
-        [Description("Checks that the SonarLint additional file is appended to the existing one")]
-        public void Roslyn_Settings_AdditionalFiles_Appended()
```

Comment:
The *AdditionalFiles* item group should be cleared so this test is no longer required. I amended the first test to check that the item group is cleared.